src/ocm/services/HostsService.ts

```diff
@@ -32,13 +34,14 @@ const HostsService = {
    * Retrieves all the hosts that are bound to the given cluster
    * @param clusterId
    */
-  async listHostsBoundToCluster(clusterId: Cluster['id']) {
+  async pollListHostsBoundToCluster(clusterId: Cluster['id']) {
     const infraEnvId = await InfraEnvsService.getInfraEnvId(clusterId);
-    const { data: hosts } = await HostsAPI.list(infraEnvId);
+    const { data: hosts } = await HostsAPI.list(infraEnvId, hostsPollingAbortController.signal);
     return hosts.filter((host) => host.clusterId === clusterId);
   },
 
   async update(clusterId: Cluster['id'], hostId: Host['id'], params: HostUpdateParams) {
+    hostsPollingAbortController.abort();
```

Comment:
It turns out that the AbortController instance can not be reused for subsequent cancellations as the signal remains cancelled once it is cancelled.
```suggestion
    hostsPollingAbortController && hostsPollingAbortController.abort();
```